man/fi_domain.3.md

```diff
@@ -261,6 +261,11 @@ interfaces enables a provider to eliminate lower-level locks.
 : A domain serialization model requires applications to serialize
   access to all objects belonging to a domain.
 
+*FI_THREAD_DOMAIN_RELAXED*
+: A domain serialization model requires applications to serialize
+  access to all objects belonging to a domain. Control interfaces
+  are not thread safe.
```

Comment:
May also add to the start of L241:  `Unless otherwise noted, ` or `Except for FI_THREAD_DOMAIN_RELAXED`